### PR TITLE
Use mail util for mails

### PR DESCRIPTION
**Pull request by aucuparia**

### CODE REVIEW

File: datarequest.py [hunk 1]
```diff
@@ -1349,10 +1259,377 @@ def mail_datarequest_submitted(ctx, researcher_email, researcher_name, request_i
...
+
+With kind regards,
+YOUth
+""".format(request_id, request_id))
+
+
+def mail_datarequest_evaluation_submit_researcher(ctx, decision, researcher_email, researcher_name,
+                                                  request_id, datamanager_email = None,
+                                                  feedback_for_researcher = None):
+    if decision == "Approved":
```
Comment (lwesterhof) [inline datarequest.py]: I think it is cleaner to have this logic outside the mail template. Make separate mail templates and move the logic one level higher.

File: datarequest.py [hunk 2]
```diff
@@ -1021,31 +956,24 @@ def api_datarequest_review_submit(ctx, data, request_id):
...
-            value = row["META_DATA_ATTR_VALUE"]
-            if name == "name":
-                researcher_name = value
-            elif name == "email":
-                researcher_email = value
+        datarequest = jsonutil.read(ctx, coll_path + "/datarequest.json")
+
+        researcher = datarequest['researchers']['contacts'][0]
+
+        researcher_name = researcher['name']
```
Comment (lwesterhof) [inline datarequest.py]: Is there any reason to not use researcher['name'] directly? They are only used in the mail call? Same for researcher_email.

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: datarequest.py (hunk 1 above)
Comment (aucuparia) [reply datarequest.py]: Agreed. Fixed.

File: datarequest.py (hunk 2 above)
Comment (aucuparia) [reply datarequest.py]: No. Fixed.

**Review by lwesterhof — APPROVED** [review]

**Pull request merged**